pkg/perfscaleanalysis/perfscalehelpers.go

```diff
@@ -0,0 +1,286 @@
+package perfscaleanalysis
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+
+	es7 "github.com/elastic/go-elasticsearch/v7"
+	"k8s.io/klog"
+)
+
+const (
+	esURL       = "http://perf-results-elastic.apps.keith-cluster.perfscale.devcluster.openshift.com"
+	esIndex     = "perf_scale_ci"
+	esQuerySize = 2000
+
+	ScaleJobsSubDir   = "scalejobs"
+	ScaleJobsFilename = "scalejobs.json"
+)
+
+// TODO: move to api pkg?
```

Comment:
Yea, I'd put this in the apis/api package if it's what we're returning to the user. If you implement the Filter interface, you'll get filtering for free too.  I think the way I did filtering is weird... I wanted to avoid reflection, but I'd be curious about your thoughts about the interface (GetColumnType, etc)